static-build/test/static-build/exports.test.lua

```diff
@@ -129,8 +129,8 @@ local check_symbols = {
 
     'ZSTD_compress',
     'ZSTD_decompress',
-    'ZSTD_free',
-    'ZSTD_malloc',
+    'ZSTD_customFree',
+    'ZSTD_customMalloc',
```

Comment:
zstd's public API is defined by `lib/zstd.h` and it has no ZSTD_free / ZSTD_customFree mention. I guess those function were added here by a mistake.

@rosik or @HustonMmmavr may know details.

Reply:
I'm not sure but seems so. 

Seems the export list wasn't generated properly. At that time we didn't develop an approach yet (https://github.com/tarantool/enterprise/blob/1.10.8-0-enterprise/src/exports_ldap.h#L3)

In this case I think it should be:

```bash
gcc -DZSTDLIB_VISIBILITY=GREPAPI  ./third_party/zstd/lib/zstd.h -E | grep '^GREPAPI'
```

```
ZSTD_compress
ZSTD_compress_usingCDict
ZSTD_compress_usingDict
ZSTD_compressBound
ZSTD_compressCCtx
ZSTD_compressStream
ZSTD_createCCtx
ZSTD_createCDict
ZSTD_createCStream
ZSTD_createDCtx
ZSTD_createDDict
ZSTD_createDStream
ZSTD_CStreamInSize
ZSTD_CStreamOutSize
ZSTD_decompress
ZSTD_decompress_usingDDict
ZSTD_decompress_usingDict
ZSTD_decompressDCtx
ZSTD_decompressStream
ZSTD_DStreamInSize
ZSTD_DStreamOutSize
ZSTD_endStream
ZSTD_flushStream
ZSTD_freeCCtx
ZSTD_freeCDict
ZSTD_freeCStream
ZSTD_freeDCtx
ZSTD_freeDDict
ZSTD_freeDStream
ZSTD_getDecompressedSize
ZSTD_getErrorName
ZSTD_getFrameContentSize
ZSTD_initCStream
ZSTD_initDStream
ZSTD_isError
ZSTD_maxCLevel
ZSTD_versionNumber
ZSTD_versionString
```

Reply:
https://github.com/tarantool/tarantool/pull/5720

Reply:
The previous list was for the old zstd. After rebase it's 

```
ZSTD_CCtx_loadDictionary
ZSTD_CCtx_refCDict
ZSTD_CCtx_refPrefix
ZSTD_CCtx_reset
ZSTD_CCtx_setParameter
ZSTD_CCtx_setPledgedSrcSize
ZSTD_compress
ZSTD_compress2
ZSTD_compress_usingCDict
ZSTD_compress_usingDict
ZSTD_compressBound
ZSTD_compressCCtx
ZSTD_compressStream
ZSTD_compressStream2
ZSTD_cParam_getBounds
ZSTD_createCCtx
ZSTD_createCDict
ZSTD_createCStream
ZSTD_createDCtx
ZSTD_createDDict
ZSTD_createDStream
ZSTD_CStreamInSize
ZSTD_CStreamOutSize
ZSTD_DCtx_loadDictionary
ZSTD_DCtx_refDDict
ZSTD_DCtx_refPrefix
ZSTD_DCtx_reset
ZSTD_DCtx_setParameter
ZSTD_decompress
ZSTD_decompress_usingDDict
ZSTD_decompress_usingDict
ZSTD_decompressDCtx
ZSTD_decompressStream
ZSTD_dParam_getBounds
ZSTD_DStreamInSize
ZSTD_DStreamOutSize
ZSTD_endStream
ZSTD_findFrameCompressedSize
ZSTD_flushStream
ZSTD_freeCCtx
ZSTD_freeCDict
ZSTD_freeCStream
ZSTD_freeDCtx
ZSTD_freeDDict
ZSTD_freeDStream
ZSTD_getDecompressedSize
ZSTD_getDictID_fromDDict
ZSTD_getDictID_fromDict
ZSTD_getDictID_fromFrame
ZSTD_getErrorName
ZSTD_getFrameContentSize
ZSTD_initCStream
ZSTD_initDStream
ZSTD_isError
ZSTD_maxCLevel
ZSTD_minCLevel
ZSTD_sizeof_CCtx
ZSTD_sizeof_CDict
ZSTD_sizeof_CStream
ZSTD_sizeof_DCtx
ZSTD_sizeof_DDict
ZSTD_sizeof_DStream
ZSTD_versionNumber
ZSTD_versionString
```